tests/utils/test_predicate_converter.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
+import pytest
+
+from kartothek.core.cube.conditions import C
+from kartothek.utils.predicate_converter import (
+    convert_predicates_to_cube_conditions,
+    write_predicate_as_cube_condition,
+)
+
+
+def test_write_predicate_as_cube_condition():
```

Comment:
You can use parametrized tests here to reduce the asserts per test. I would also suggest to separate the `test_raises_when...` into separate tests.